sematic/api/endpoints/meta.py

```diff
@@ -51,3 +57,59 @@ def env_endpoint(user: Optional[User]) -> flask.Response:
             continue
 
     return flask.jsonify(dict(env=env))
+
+
+@sematic_api.route("/api/v1/meta/health", methods=["GET"])
+def get_server_health() -> flask.Response:
+    """Get information about the server health.
+
+    Returns
+    -------
+    A flask response with a json payload. The payload has two fields:
+        api: status info about basic access to the API
+        db: status info about access to the DB
+    Status info for each of these fields is a dict of the form:
+    {
+        "healthy": <bool>,
+        "message": <str message about status>,
+    }
+    If any of the "healthy" fields is `False`, will return a 503 status code.
+    """
+    # healthy by nature of the fact that this API is working.
+    api_status = dict(
+        healthy=True,
+        message="API is healthy",
+    )
+
+    db_healthy = True
+    db_message = "Basic database access verified"
+    with db().get_session() as session:
+        try:
+            statement = sqlalchemy.text("SELECT id FROM runs LIMIT 1;")
+            if len(list(session.execute(statement))) > 0:
+                db_message = "Verified at least one run is present in the DB."
```

Comment:
This will fail on fresh deployments / installations, preventing the service from ever becoming available.
Please accept any non-error response as healthy, or introduce a pre-populated table that just serves as a health probe.

Reply:
Note that db_healthy is set to True even if there are 0 run ids returned. The unit test even tests that case.

Reply:
then perhaps this message isn't quite accurate?  should maybe say "Verified that DB runs can be queried."